usage_guide.md

```diff
@@ -0,0 +1,5 @@
+
+
+TCP
```

Comment:
Hm, what is this doc? Was it intended to be included in this PR?

Reply:
Nope! Not sure how that got in there ... Thanks for pointing that out.